README.md

```diff
@@ -64,6 +64,20 @@ Be aware of a few limitations:
 - a line that can take one of two branches gets marked as covered even if only one branch is tested
 - currently, code run by Julia's internal interpreter [is not marked as covered](https://github.com/JuliaLang/julia/issues/37059).
 
+### Exclude specific lines or sections from coverage
+
+To exclude specific sections, wrap the section in `COV_EXCL_START` - `COV_EXCL_STOP` blocks:
+```julia
+# COV_EXCL_START
+foo() = nothing
+# COV_EXCL_STOP
+```
+
+To exclude a single line, add `COV_EXCL_LINE` anywhere:
```

Comment:
"Anywhere"? Perhaps "add a comment with COVL_EXCL_LINE on that line"?

Reply:
Anywhere was for the `occursin` https://github.com/JuliaCI/CoverageTools.jl/blob/9d8fba371f4c65a7e9ca3bdaae753a52bcb22218/src/CoverageTools.jl#L212

Reply:
I mean:
```julia
const #=COVL_EXCL_LINE=# a = 1
```
would also work, as awkward as it may be.